cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SchemaIndexTest.scala

```diff
@@ -267,7 +289,7 @@ class SchemaIndexTest extends DocumentingTestBase with QueryStatisticsTestSuppor
         "However, the query does not need to have equality on all properties. It can have ranges and existence predicates as well. " +
         "But in these cases rewrites might happen depending on which properties have which predicates, " +
         "see <<administration-indexes-single-vs-composite-index, composite index limitations>>. " +
-        "The following query will use the composite index defined <<administration-indexes-create-a-composite-index, earlier>>: ",
+        "The following query will use the composite index defined <<administration-indexes-create-a-composite-index-for-nodes, earlier>>: ",
```

Comment:
I think it would be good to also add some relationship examples here, i.e. at https://neo4j.com/docs/cypher-manual/4.3-preview/query-tuning/indexes/. Perhaps you can do it in collaboration with the planner team?

Reply:
I don't know how done the planner team is with their part, but I'd expect that and hints to be their to document (but yes we should probably make sure they know that as well)

(I don't know how they will be used, I just create, drop and list them :P)